modules/distr-tracing-accessing-jaeger-console.adoc

```diff
@@ -0,0 +1,60 @@
+////
+Module included in the following assemblies:
+
+////
+
+[id="distr-tracing-accessing-jaeger-console_{context}"]
+= Accessing the Jaeger console
+
+To access the Jaeger console you must have either {ProductName} or {DTProductName} installed, and {JaegerName} installed and configured.
+
+The installation process creates a route to access the Jaeger console.
+
+If you know the URL for the Jaeger console, you can access it directly.  If you do not know the URL, use the following directions.
+
+.Procedure from OpenShift console
+. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Navigate to *Networking* -> *Routes*.
+
+. On the *Routes* page, select the control plane project, for example `tracing-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+. If necessary, use the filter to find the `jaeger` route.  Click the route *Location* to launch the console.
+
+. Click *Log In With OpenShift*.
+
+////
+.Procedure from Kiali console
+
+. Launch the Kiali console.
+
+. Click *Distributed Tracing* in the left navigation pane.
+
+. Click *Log In With OpenShift*.
+////
+
+.Procedure from the CLI
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you use Red Hat OpenShift Dedicated, you must have an account with the `dedicated-admin` role.
++
+[source,terminal]
+----
+$ oc login https://<HOSTNAME>:6443
+----
++
+. To query for details of the route using the command line, enter the following command. In this example, `tracing-system` is the control plane namespace.
++
+[source,terminal]
+----
+$ export JAEGER_URL=$(oc get route -n tracing-system jaeger -o jsonpath='{.spec.host}')
+----
++
+. Launch a browser and navigate to ``\https://<JAEGER_URL>``, where `<JAEGER_URL>` is the route that you discovered in the previous step.
+
+. Log in using the same user name and password that you use to access the {Product-title} console.
+
+. If you have added services to the service mesh and have generated traces, you can use the filters and *Find Traces* button to search your trace data.
++
+If you are validating the console installation, there is no trace data to display.
```

Comment:
I am not sure what this sentence wants to say.

Reply:
Jaeger deployment should show traces for `jaeger-query` if pace is refreshed - jaeger traces itself.

Reply:
Changed to "If you are validating the console installation, there is no data to display until you start collecting traces."